docs/reference/Parsers/Regex/Regex/index.html

```diff
@@ -30,13 +30,13 @@
 <pre>	
 Io> re := "is.*a" asRegex
 Io> "This is a test. This is also a test." \
-    allMatchesOfRegex(" is[^.]*a") replaceAllWith(" is not a")
+    matchesOfRegex(" is[^.]*a") replaceAllWith(" is not a")
 ==> "This is not a test. This is not a test.
 </pre>
 
 <h4>Example 2</h4>
 <pre>	
-Io> "11aabb" allMatchesOfRegex("aa*")
+Io> "11aabb" matchesOfRegex("aa*")
 ==> list("a", "a")
```

Comment:
Could this be an accidental change from a1153f2 / #139?  I left it in just in case.